### ISSUE TITLE
Accessibility improvements to authentication pages

### ISSUE THREAD

**Issue by echappen** [ustaxcourt/ef-cms#592]
Applies to:
- Sign Up page
- Sign In page
- Forgot Password flow

## 1. Logo has insufficient alt text
Alt text for Us Tax Court logo is currently "logo." Since informational text is present in the logo and not in the markup of the page, the alt text should convey this information. 

Additionally, this logo is meant to represent the name of the app or product on the page. Without sufficient alt text or having the name elsewhere on the page, it's ambiguous what exactly you're signing in to.

#### Solution
Change alt text of logos to "United States Tax Court."

#### Why It Matters
If the alternative text for an image does not provide the same content or information conveyed by the image, that content will not be available to screen reader users and when images are unavailable.

Ensure that the alternative text for the image or image input provides a succinct, yet equivalent alternative to the content and function of the image. Screen readers and browser presentation inform the user that the object is an image, so alternative text of "image of..." (and similar) should be avoided. If the image does not convey content or if the content is presented in nearby text (e.g., a caption), null/empty alternative text (alt="") is appropriate.

[More info here](https://www.hook42.com/blog/a11y-tips-how-make-your-logo-accessible)

#### Why It Matters
Headings (`<h1>`-`<h6>`) provide important document structure, outlines, and navigation functionality to assistive technology users.

#### How to Fix It
Provide a clear, consistent heading structure, generally one `<h1>` and sub-headings as appropriate. Except for very simple pages, most web pages should have a heading structure.

## 3. Password requirements do not have sufficient contrast ratio
Text in green has an insufficient contrast ratio of less than 4.5:1, the threshold for small text.

### Note
Since Authentication pages are managed through Amazon Cognito, page editing may be limited. [Info on how to edit Cognito pages](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-app-ui-customization.html)

Issues caught using [WAVE Evaluation plugin](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) for Chrome

**Comment by JessicaMarine** [comment]
Moved to Flexion: https://app.zenhub.com/workspaces/flexionef-cms-5bbe4bed4b5806bc2bec65d3/issues/flexion/ef-cms/8107
